js_modules/dagit/packages/core/src/instance/backfill/BackfillPage.tsx

```diff
@@ -0,0 +1,310 @@
+import {gql, useQuery} from '@apollo/client';
+import {Page, PageHeader, Colors, Box, Tag, Table, Spinner} from '@dagster-io/ui';
+import dayjs from 'dayjs';
+import duration from 'dayjs/plugin/duration';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import React from 'react';
+import {Link, useParams} from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+import {PYTHON_ERROR_FRAGMENT} from '../../app/PythonErrorFragment';
+import {PythonErrorInfo} from '../../app/PythonErrorInfo';
+import {useTrackPageView} from '../../app/analytics';
+import {BulkActionStatus, RunStatus} from '../../graphql/types';
+import {useDocumentTitle} from '../../hooks/useDocumentTitle';
+import {RunFilterToken, runsPathWithFilters} from '../../runs/RunsFilterInput';
+
+import {
+  BackfillStatusesByAssetQuery,
+  BackfillStatusesByAssetQueryVariables,
+} from './types/BackfillPage.types';
+
+dayjs.extend(duration);
+dayjs.extend(relativeTime);
+
+export const BackfillPage = () => {
+  const {backfillId} = useParams<{backfillId: string}>();
+  useTrackPageView();
+  useDocumentTitle(`Backfill | ${backfillId}`);
+
+  const {data, loading} = useQuery<
+    BackfillStatusesByAssetQuery,
+    BackfillStatusesByAssetQueryVariables
+  >(BACKFILL_DETAILS_QUERY, {
+    variables: {backfillId},
+  });
+
+  console.log({backfillId, data, loading});
+
+  const backfill = data?.partitionBackfillOrError;
+
+  function content() {
+    if (!backfill || loading) {
+      return (
+        <Box padding={64}>
+          <Spinner purpose="page" />
+        </Box>
+      );
+    }
+    if (backfill.__typename === 'PythonError') {
+      return <PythonErrorInfo error={backfill} />;
+    }
+
+    function getRunsUrl(status: 'requested' | 'complete' | 'failed' | 'targeted') {
+      const filters: RunFilterToken[] = [
+        {
+          token: 'tag',
+          value: `dagster/backfill=${backfillId}`,
+        },
+      ];
+      switch (status) {
+        case 'requested':
+          filters.push(
```

Comment:
I'm not sure about how to map these statuses tbh @bengotow.

Reply:
I think this looks good!